rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/condition/DataRuleGeneratorTest.xtend

```diff
@@ -81,8 +81,8 @@ class DataRuleGeneratorTest {
 							}
 							
 							String failureMessage = result.getError();
-							if (failureMessage == null) {
-								failureMessage = "Condition " + NAME + " failed.";
+							if (failureMessage == null || failureMessage.contains("Null") || failureMessage == "") {
```

Comment:
Same here.

Reply:
this is to factor in the Null operand error message to  just return `Condition has failed` instead of wired messages

Reply:
`EMIR_VR_2023_03:- Null operand null ExecutionTimestamp 2018-04-01T14 15 36Z`

Reply:
`EMIR_VR_2060_01:- Null operand null Integer 0 and AssetClass INTR does not equal String EQUI and AssetClass INTR does not equal String COMM
Null operand: [[] : null] >= [[Integer] : 0]`

Reply:
for error message like this it would return `EMIR_VR_2060_01:- Condition has failed`